docs/docs/configuration.md

```diff
@@ -52,6 +52,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.bloom-filter-enabled.column.col1       | (not set)                   | Hint to parquet to write a bloom filter for the column: 'col1'                                                                                                                                    |
 | write.parquet.bloom-filter-max-bytes                 | 1048576 (1 MB)              | The maximum number of bytes for a bloom filter bitset                                                                                                                                             |
 | write.parquet.bloom-filter-fpp.column.col1           | 0.01                        | The false positive probability for a bloom filter applied to 'col1' (must > 0.0 and < 1.0)                                                                                                        |
+| write.parquet.stats-enabled.column.default           | true                        | Default flag to enable parquet column statistics for all columns in the table                                                                                                                     |
```

Comment:
Just realized, that we can't set the config value for a column which is called `default`.
Maybe:
`write.parquet.stats-enabled-default`

Also for the description:
`Controls whether to collect parquet column statistics when not specified on column level`

Reply:
Good catch. It should be write.parquet.stats-enabled.default, following https://iceberg.apache.org/docs/latest/configuration/#write-properties, write.metadata.metrics.default. 
I will upload a new patch which addresses all review comments today.